### ISSUE TITLE
Info tabs linking to route

### ISSUE THREAD

**Issue by Rowcii** [pokedextracker/pokedextracker.com#326]
Hey!

I'm using the site for my second living dex on USUM now, and I spend a lot of time looking on bulbapedia where the actual pokemon is in the specified route.

Can this be changed by either:

- Making the route a hyperlink

- Add a button on the bottom next to Bulbapedia and Serebii

Thanks for considering :)

**Comment by ami** [comment]
thanks @Rowcii - this is a great suggestion!

internal note - we will probably need to combine this with an account setting of "preferred wiki source" or something like that so we can default the links to either serebii or bulbapedia depending on the account's preference.

**Comment by Rowcii** [comment]
> 
> 
> thanks @Rowcii - this is a great suggestion!
> 
> internal note - we will probably need to combine this with an account setting of "preferred wiki source" or something like that so we can default the links to either serebii or bulbapedia depending on the account's preference.


I'm not so sure about the account settings option :o
I think the routes should always be clickable (when you click on the little info icon next to a pkmn) 
Let's say for example you wanna catch a Caterpie in USUM there is a specific bush you need to look for. With the current version of the dex tracker you need to> Press the 'I' > Go to Bulbapedia > Search for the "Game location" button > Again look for Caterpie. I think this is too many things to click on. So making the "Route 1" link to the Game locations page of Caterpie would be much nicer for everyone. Not just for people who have it enabled. I didn't even know dextracker had account settings until this :D

**Comment by ami** [comment]
@Rowcii yes, the links would definitely always be clickable! what i meant was that there would be an account setting where you can pick if you want your links to go to bulbapedia, serebii, pokemondb, etc. :)
